skills/declarative/tests/common/Microsoft.Bot.Dialogs.Tests.Common.csproj

```diff
@@ -6,10 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.13.0-rc2" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive" Version="4.13.0-rc2" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive.Testing" Version="4.13.0-rc2.preview" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Declarative" Version="4.13.0-rc2" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.13.0-rc4" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive" Version="4.13.0-rc4" />
```

Comment:
We may want to just break this out into a property in the msbuild so there is only one place to update things. It can be per csproj or have a common .props file included with the csproj that can be referenced.

Reply:
Agreed, that would probably make sense. Let's revisit this as I don't think it's urgent for RC3 just yet. I'll find some time and we can see what we can do to smooth out the process.